sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -427,10 +470,10 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
       pluginContext: PluginContext,
       extraConf: java.util.Map[String, String]): Unit = {
     try {
-      if (Cuda.getComputeCapabilityMajor < 6) {
-        throw new RuntimeException(s"GPU compute capability ${Cuda.getComputeCapabilityMajor}" +
-          " is unsupported, requires 6.0+")
-      }
+      // Checks if the current GPU architecture is supported by the
+      // spark-rapids-jni and cuDF libraries.
+      RapidsPluginUtils.checkGpuArchitecture()
```

Comment:
Given the long debate on how to do this properly, do we want an "escape valve" internal config to skip this check on the off-chance there's a GPU arch in the future that we want to work, and otherwise would work, but this check gets it wrong and prevents startup?